api/core/model_runtime/model_providers/wenxin/llm/ernie-4.0-8k.yaml

```diff
@@ -15,16 +15,17 @@ parameter_rules:
     default: 0.8
   - name: top_p
     use_template: top_p
-  - name: max_tokens
-    use_template: max_tokens
-    required: true
+  - name: max_output_tokens
+    use_template: max_output_tokens
     default: 256
     min: 1
     max: 4800
-  - name: presence_penalty
-    use_template: presence_penalty
+  - name: penalty_score
+    use_template: penalty_score
```

Comment:
there is not a template named penalty_score in our default templates, you can refer to `core.model_runtime.entities.defaults`